core/domain/stats_jobs_continuous_test.py

```diff
@@ -66,13 +66,19 @@ class InteractionAnswerSummariesAggregatorTests(test_utils.GenericTestBase):
     ALL_CC_MANAGERS_FOR_TESTS = [MockInteractionAnswerSummariesAggregator]
 
     def _record_start(self, exp_id, exp_version, state_name, session_id):
+        """Calls StartExplorationEventHandler and starts recording the
+        exploration events corresponding to the given exploration id.
+        """
         event_services.StartExplorationEventHandler.record(
             exp_id, exp_version, state_name, session_id, {},
             feconf.PLAY_TYPE_NORMAL)
 
     def _get_calc_output_model(
             self, exploration_id, state_name, calculation_id,
             exploration_version=stats_jobs_continuous.VERSION_ALL):
+        """Gets the StateAnswersCalcOutputModel corresponding to the given
```

Comment:
Gets ->  Returns ?

Reply:
Ditto as above.

Reply:
Gets is fine I believe, and consistent with the function name. That's why I suggested gets :)